src/plugins/documenter.jl

```diff
@@ -121,6 +138,7 @@ view(p::Documenter, t::Template, pkg::AbstractString) = Dict(
     "PKG" => pkg,
     "REPO" => "$(t.host)/$(t.user)/$pkg.jl",
     "USER" => t.user,
+    "DEVBRANCH" => something(p.devbranch, default_branch(t)),
```

Comment:
```suggestion
    "BRANCH" => p.devbranch === nothing ? default_branch(t) : p.devbranch,
```

I'm just calling this `BRANCH` in all the other views that need it for consistency.
Also, `something` will error if both are `nothing`, but it's actually not a problem.